src/EditorFeatures/Core.Wpf/InlineRename/UI/Dashboard/RenameDashboard.xaml.cs

```diff
@@ -325,6 +329,7 @@ private async Task CommitAsync()
         {
             try
             {
+                using var token = _listener.BeginAsyncOperation(nameof(CommitAsync));
```

Comment:
this is not a fire-and-forget async work.  this doesn't need a token to track it (The Task returned tracks it) :)

Reply:
I was thinking you want to set up a token to track all the rename op? 🤔

Reply:
Anyway, I moved the token to caller